recipes-connectivity/iproute2/iproute2_4.%.bbappend

```diff
@@ -1,10 +1,7 @@
-PR .= ".1"
+PR .= ".2"
 
 PACKAGE_ARCH = "${MACHINE_ARCH}"
-EXTRA_OEMAKE += "-j 1"
```

Comment:
I was doubtful it would be a problem so I tried in different trees and had no issue. Also, this is appending a recipe upstream which is assumed to build using multiple threads.